src/explore-education-statistics-common/src/modules/table-tool/components/TimePeriodDataTable.tsx

```diff
@@ -421,7 +421,7 @@ const TimePeriodDataTable = forwardRef<HTMLElement, Props>(
       onError?.(isErrorLike(error) ? error.message : 'Unknown error');
 
       return (
-        <WarningMessage>
+        <WarningMessage testId="permalink-render-error">
```

Comment:
`TimePeriodDataTable` is a generic component that is used in lots of different contexts, we shouldn't use an ID like this as it would appear everywhere.

Would probably go with `table-error` or something similar